shell/assets/translations/en-us.yaml

```diff
@@ -1566,6 +1566,9 @@ cluster:
       label: Roles
     labels:
       label: Kubernetes Node Labels
+    noAccessBanner: "You do not have access to see this machine pool's configuration."
```

Comment:
This whole file is out of data with master - it has over-written the latest string with old ones - can you rebase again, or revert back

Reply:
Should be fixed. Thanks for the feedback